packager/gitpackaging.py

```diff
@@ -100,9 +101,11 @@ def update_changed_artifacts(git_repos, refs, min_versions, outdir):
     for artifact in added:
         logger.debug("New artifact %s version %s", artifact.addon_id, artifact.version)
         dest = os.path.join(outdir, artifact.addon_id)
+        original_branch = artifact.treeish.split(':')[0].split('/')[-1]
+        linkdir = os.path.join('..', '..', original_branch, artifact.addon_id)
```

Comment:
You can't assume this. The whole point of adding the concept of 'targets' was to avoid this; i.e. couple the directory layout and the git branch layout.